### ISSUE TITLE
Error wheb editing unvetted proposal directly after creating it

### ISSUE THREAD

**Issue by amassarwi** [decred/politeiagui#1976]
Currently on test-proposals I get a runtime error when trying to edit an unvetted proposal directly after creating it.

I tried to reproduce on master and it seems like it was fixed, decided to open an issue anyway, in case we encounter it in the future.

**Comment by thi4go** [comment]
I got this error as well, when trying to edit right after a proposal submission, on the proposal detail page you get redirected to when submitting.

![Screenshot from 2020-06-04 12-09-55](https://user-images.githubusercontent.com/2729122/83774738-777b6900-a65c-11ea-947a-a3cb7b21a698.png)

**Comment by amassarwi** [comment]
That looks different than the won I got tbh!
@thi4go  anything in the console? is it on test-proposals as well ?

**Comment by amassarwi** [comment]
Yep, it's the same error, I think it's related to the response we get from BE on success & I guess it has to do with state after dispatching a success action with the proposal details.

**Comment by tiagoalvesdulce** [comment]
I still can't repro this issue. Can you guys track every steps you do from log in to the proposal edition so I can try to repro it on my end?

**Comment by lukebp** [comment]
I just got a runtime type error on test-proposals.  It was when submitting a RFP submission.  The first submission failed due to an invalid image.  The next attempt was successful but the gui threw a type error.
